### PR TITLE
Clever release blog post

### PR DESCRIPTION
**Pull request by benvillalobos**
Blog post about adding MakeCode to the [Clever](https://clever.com) platform.

### CODE REVIEW

**Comment by benvillalobos** [comment]
Is there an equivalent to `pxt uploadtarget` that I can do here to see how the blog post will look? I'm a little worried the screenshots might be too big.

File: docs/blog/clever-release.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# MakeCode Clever Integration Released!
+Just in time for the fall semester, MakeCode has integrated with [Clever](https://www.clever.com/)!
```
Comment (Jaqster) [inline docs/blog/clever-release.md:2]: I might say something like "Just in time for back to school, MakeCode is adding more ways for students and teachers to sign in!  [Clever](https://www.clever.com) is an education platform that provides identity and security services, making it easy to access classroom resources all in one place."

File: docs/blog/clever-release.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Today, you can find our apps [Microsoft MakeCode Arcade](https://clever.com/library/app/makecode-arcade) and [Microsoft MakeCode for micro:bit](https://clever.com/library/app/makecode-microbit) by searching for it in the [Clever Library](https://clever.com/library).
```
Comment (Jaqster) [inline docs/blog/clever-release.md:4]: "searching for them"

File: docs/blog/clever-release.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+![MakeCode Arcade in the Clever Library](/docs/static/blog/clever-release/arcade-app-clever.png)
+
+Authorizing our app means your entire classroom gains instant access to MakeCode Arcade and MakeCode for micro:bit through their Clever account.
+
+Because we store projects and progress securely in the cloud, students can easily start working in the classroom and can continue at home! To do so, use the fancy new Clever button when signing in to our sites.
+
+![Clever Login Button](/docs/static/blog/clever-release/clever-login-button.png)
+
+If you can't find our apps in the Clever Library, it's possible you're in a district that allows apps for classroom use on a case-by-case basis. Please contact your district admins for help with allowing our apps within your district. For any issues you run into, please reach out to [@MSMakeCode](https://twitter.com/MSMakeCode), make a post in our forum's [Educator Lounge](https://forum.makecode.com/c/educators-lounge/24) in our forum, or [file a bug on GitHub](https://github.com/microsoft/pxt-arcade/issues/new/choose).
```
Comment (Jaqster) [inline docs/blog/clever-release.md:14]: "...district IT administrator to enable the MakeCode apps in the Clever portal"
"...post to the [Educator Lounge]..."

**Review by Jaqster — APPROVED** [review]
Looks great Ben!  Just had some small wording suggestions.  Thanks for posting!  Excited to get the word out....

**Review by abchatra — APPROVED** [review]

**Comment by abchatra** [comment]
@ganicke if you can please give a quick review as well.

**Review by ganicke — APPROVED** [review]

**Comment by Jaqster** [comment]
Can we merge this?

**Comment by benvillalobos** [comment]
Just came back from OOF, thanks @Jaqster for the suggestions, @ganicke for the edits, and @abchatra for merging! 🥳

**Pull request merged**